AmplifyPlugins/Predictions/AWSPredictionsPlugin/Dependency/AWSTranscribeStreamingAdapter.swift

```diff
@@ -131,17 +131,17 @@ class AWSTranscribeStreamingAdapter: AWSTranscribeStreamingBehavior {
                             continuation.yield(transcribedPayload)
                             let isPartial = transcribedPayload.transcript?.results?.map(\.isPartial) ?? []
                             let shouldContinue = isPartial.allSatisfy { $0 }
-                            return shouldContinue
+                            return shouldContinue ? .continueToReceive : .stopAndInvalidateSession
```

Comment:
are the changes in file intentional or related?

Reply:
these are downstream changes as result type changed from `Bool` to `WebSocketSession.WebSocketMessageResult` in `WebSocketSession` - `WebSocketSession` is used in `AWSTranscribeStreamingAdapter` as well